decidim-comments/app/frontend/comments/add_comment_form.component.jsx

```diff
@@ -228,6 +229,7 @@ const AddCommentFormWithMutation = graphql(gql`
             if (comment.id === commentableId) {
               return {
                 ...comment,
+                hasReplies: true,
```

Comment:
I don't understand this:

- First, why do you need a flag for this, can you just check if there are replies?
- If this is needed, why is set to true here?